### ISSUE TITLE
Cant recreate default certificates

### ISSUE THREAD

**Issue by ekremparlak** [TeaSpeak/TeaSpeak#584]
**Describe the bug**
If you remove the certificates, cant create again

**To Reproduce**
1. Go to 'certs folder and remove the files'
2. 'restart server'

**Expected behavior**
Able to create default certificates.

**Logs**
```
[2019-03-22 16:26:57] [DEBUG] Log successfully started!
[2019-03-22 16:26:57] [INFO]    GEN | Starting TeaSpeak-Server v1.3.11-beta [Build: 1553204177]
[2019-03-22 16:26:57] [INFO]    GEN | Starting music providers
[2019-03-22 16:26:57] [DEBUG] [Music] Provider load order:
[2019-03-22 16:26:57] [DEBUG] [Music] [0] providers/000ProviderFFMpeg.so
[2019-03-22 16:26:57] [DEBUG] [Music] [1] providers/001ProviderYT.so
[2019-03-22 16:26:57] [DEBUG] [Music] [FFMPEG] Using config file located at providers/config_ffmpeg.ini
[2019-03-22 16:26:57] [DEBUG] [Music] [FFMPEG] Missing configuration file. Using default values
[2019-03-22 16:26:57] [DEBUG] [Music] [FFMPEG] Executing command "ffmpeg -version"
[2019-03-22 16:26:57] [ERROR] [Music] [FFMPEG] Could not find ffmpeg (Error: "sh: 1: ffmpeg: not found")
[2019-03-22 16:26:57] [ERROR] [Music] [FFMPEG] How to download/install ffmpeg: "sudo apt-get install ffmpeg"
[2019-03-22 16:26:57] [ERROR] [Music] Could not create music provider for providers/000ProviderFFMpeg.so
[2019-03-22 16:26:57] [ERROR] [Music] Could not load music provider providers/001ProviderYT.so. Error: 000ProviderFFMpeg.so: cannot open shared object file: No such file or directory
[2019-03-22 16:26:59] [DEBUG]   GEN | Created a new async thread pool!
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 1 - Guest Server Query
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 2 - Admin Server Query
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 3 - Server Admin
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 4 - Normal
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 5 - Guest
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 6 - Channel Admin
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 7 - Operator
[2019-03-22 16:26:59] [DEBUG]   GEN | Push back group -> 8 - Guest
[2019-03-22 16:26:59] [DEBUG] GLOBL | Instance admin group id 2
[2019-03-22 16:26:59] [DEBUG] GLOBL | Server guest group id 1
[2019-03-22 16:26:59] [DEBUG] GLOBL | Server music group id 1
[2019-03-22 16:26:59] [INFO]    GEN | Got 4 saved channels
[2019-03-22 16:26:59] [DEBUG]   GEN | Deleting old bans
[2019-03-22 16:26:59] [INFO]   FILE | Starting server on 0.0.0.0:30303
[2019-03-22 16:26:59] [INFO]  QUERY | Starting server on 0.0.0.0:10101
[2019-03-22 16:26:59] [CRITICAL] GLOBL | Failed to initialize WebClient certificate! (Could not read certificate: 139984863058120:error:0900006e:PEM routines:OPENSSL_internal:NO_START_LINE:/workspace/TeaSpeak-Parent_master-FRUL5U2YEGMIHINML2BZCK6KBI6I4EHNXBFX3JFUSZG2NLVUAXHA@2/libraries/boringssl/crypto/pem/pem_lib.c:622:Expecting: CERTIFICATE
[2019-03-22 16:26:59] [CRITICAL] )
[2019-03-22 16:26:59] [INFO]    GEN | Stopping application
[2019-03-22 16:26:59] [DEBUG] GLOBL | Stopping all virtual servers
[2019-03-22 16:26:59] [DEBUG] GLOBL | All virtual server stopped
[2019-03-22 16:26:59] [DEBUG] QUERY | Stopping query server
[2019-03-22 16:26:59] [DEBUG] QUERY | Event base executed (0x7f50c4087c00)
[2019-03-22 16:26:59] [DEBUG] QUERY | Event base terminated (0x7f50c4087c00)
[2019-03-22 16:26:59] [DEBUG] QUERY | Query server stopped
[2019-03-22 16:26:59] [DEBUG]  FILE | Stopping file server
[2019-03-22 16:26:59] [DEBUG]  FILE | File-Server accept thread terminated
[2019-03-22 16:26:59] [DEBUG]  FILE | File server stopped
[2019-03-22 16:27:00] [INFO]    GEN | Application suspend successful!
```

**Environment information:**
 Operating system: (`Ubuntu 18.04 x64`)
 TeaSpeak version: [Latest]
 I am using `experimental_31`: (no)

**Additional context**
When restarting to server it create the file but they are empy
```
-rw-rw-r-- 1 teaspeak teaspeak    0 Mar 22 16:26 default_certificate.pem
-rw-rw-r-- 1 teaspeak teaspeak    0 Mar 22 16:26 default_privatekey.pem
-rw-rw-r-- 1 teaspeak teaspeak  969 Mar 22 16:26 query_certificate.pem
-rw-rw-r-- 1 teaspeak teaspeak 1704 Mar 22 16:26 query_privatekey.pem
```

**Comment by WolverinDEV** [comment]
What happens if you remove these "empty" certs again?
Does the server has write permissions within the folder (I guess, but Its better to ensure)

**Comment by ekremparlak** [comment]
> What happens if you remove these "empty" certs again?
Server create same empty certificates again.
Does the server has write permissions within the folder (I guess, but Its better to ensure)
`drwxr-xr-x 2 teaspeak teaspeak     4096 Mar 24 14:20 certs`

**Comment by WolverinDEV** [comment]
Hmm yeahr I'll take a look.
In general the way how the certs work for the web client will change.
Within the next version SNI will be supported (Server name indication). This means multible certs for multible domains :)

**Comment by WolverinDEV** [comment]
Hmm not sure anymore, but its def. fixed in 1.3.13 (Wirh SNI then)
